### DISCUSSION TITLE
complejidad FESTIVAL y HAPPINESS BUFF

### DISCUSSION THREAD

**Discussion by ilpeter** [IIC2133-PUC/2024-2#118] [Tarea 1] [answered]
quería saber si la complejidad de HAPPINESS _BUFF es correcta O(n + log(s)) ya que con algunos compañeros pensamos que la mínima es O(n + log(n)). También saber si la complejidad de FESTIVAL es correcta ya que con solo recorrer los festivales se logra O(s) (que es la complejidad pedida) por lo que encontrar mínimo, maximo y mediana debería ser O(1) lo que no sabría como abarcar. Gracias!

**Comment by 4n1t4** [comment, marked as answer]
Happiness es correcto O(n+log(s))
Festival tambien es correcta: O(s)
